web-animations-1/Overview.bs

```diff
@@ -3799,14 +3805,14 @@ if <em>all</em> of the following conditions are true:
     [=timeline=].
 *   The [=animation=] has an associated [=target effect=].
 *   The [=target effect=] associated with the [=animation=] is [=in effect=].
-*   The [=target effect=] has an associated [=target element=].
+*   The [=target effect=] has an associated [=effect target=].
```

Comment:
@birtles for your thoughts here

Reply:
I agree this is akward, your solution looks good to me.

Reply:
I think `effect` is too vague a term for the model (it's fine for the API though). Any old key frame effect is an effect so this can easily get confusing. We mean to say the particular effect that the animation is playing.

I agree we should keep the term `effect target` but rename `target effect` to something else.

At one point I think we called the `target effect` the `source effect` but that's slightly inaccurate. An animation _drives_ an effect, but is quite happy to keep ticking along without one. So it's not really the "source" for anything.

`client effect`? `applied effect`? `associated effect`?

I think I like "associated effect" best?

Reply:
`associated effect` SGTM. It's somewhat vague, but good enough I think.